src/summarize.jl

```diff
@@ -25,15 +25,33 @@ end
 
 function Base.show(io::IO, ::MIME"text/plain", df::ChainDataFrame)
     digits = get(io, :digits, 4)
-    formatter = PrettyTables.ft_printf("%.$(digits)f")
-
-    println(io, df.name)
-    # Support for PrettyTables 0.9 (`borderless`) and 0.10 (`tf_borderless`)
-    PrettyTables.pretty_table(
-        io, df.nt;
-        formatters = formatter,
-        tf = isdefined(PrettyTables, :borderless) ? PrettyTables.borderless : PrettyTables.tf_borderless,
-    )
+    if pkgversion(PrettyTables) < v"3.0"
```

Comment:
```suggestion
    @static if pkgversion(PrettyTables) < v"3.0"
```

I'm not sure it really makes a difference, but it probably can't hurt.

Reply:
`pkgversion` is not supported on Julia < 1.9. IMO the easiest (for the compiler and for maintenance) would be to just require PrettyTables 3 and not introduce any code that is conditional on the loaded PrettyTables version.

Reply:
This is causing issues for me right now unfortunately, because pkgversion doesn't work in our systemimage build   because pkgversion returns nothing.